Gemfile.lock

```diff
@@ -2,79 +2,80 @@ GEM
   remote: https://rubygems.org/
   specs:
     SyslogLogger (2.0)
-    actionmailer (4.2.0)
-      actionpack (= 4.2.0)
-      actionview (= 4.2.0)
-      activejob (= 4.2.0)
+    actionmailer (4.2.5)
+      actionpack (= 4.2.5)
+      actionview (= 4.2.5)
+      activejob (= 4.2.5)
```

Comment:
Yes, please. Updating all gems is a very big change and could cause a lot of regressions.